### DISCUSSION TITLE
Los mensajes antiguos de canales user=<userId> no se cargan

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#155] [4. Bug Reports] [answered]
Buenas,

El bug es que los mensajes que se cargan al entrar a un chat de usuario con url 
https://web.cotalker.com/c/contacts?user=<userId> no se cargan. Pareciera que la conversación es un chat nuevo.

Saludos.

**Comment by acedward** [comment]
Duplicado de https://github.com/Cotalker/documentation/discussions/148

**Comment by seiturra** [comment, marked as answer]
@acedward una acotación. La 148 es sobre cuando recién se entra a una cuenta y no permite enviar mensajes. 
Lo que estoy reportando no es lo mismo pero probablemente está relacionado. Es que se pueden enviar mensajes pero no se cargan los mensajes anteriores.

**Reply by acedward** [reply]
Si, podrían ser efectivamente dos bugs - pero los dos están siendo corregidos. Fue catalogado de alta prioridad
Está en desarrollo la solución!
